File: JAVA/Aksesoris.java
class Aksesoris extends PetShop
{
  private String jenis;
  private String bahan;
  private String warna;

  // konstruktor
  public Aksesoris(){

  }

  public Aksesoris(int id, String nama, int harga, int stok, String jenis, String bahan, String warna){
    super(id, nama, harga, stok);
    this.jenis = jenis;
    this.bahan = bahan;
    this.warna = warna;
  }

  // getter & setter
  public String getJenis(){
    return jenis;
  }

  public void setJenis(String jenis){
    this.jenis = jenis;
  }

  public String getBahan(){
    return bahan;
  }

  public void setBahan(String bahan){
    this.bahan = bahan;
  }

  public String getWarna(){
    return warna;
  }

  public void setWarna(String warna){
    this.warna = warna;
  }
}

File: PYTHON/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
    jenis = ""
    bahan = ""
    warna = ""
    
    def __init__(self, idProduk=0, nama="", harga=0, stok=0, jenis="", bahan="", warna=""):
        super().__init__(idProduk, nama, harga, stok)  # Memanggil konstruktor PetShop
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    def setJenis(self, jenis): self.jenis = jenis
    def getJenis(self): return self.jenis

    def setBahan(self, bahan): self.bahan = bahan
    def getBahan(self): return self.bahan

    def setWarna(self, warna): self.warna = warna
    def getWarna(self): return self.warna  
